extension/src/plots/webview/messages.ts

```diff
@@ -112,6 +112,9 @@ export class WebviewMessages {
       case MessageFromWebviewType.TOGGLE_EXPERIMENT:
         return this.setExperimentStatus(message.payload)
       case MessageFromWebviewType.ZOOM_PLOT:
+        if (message.payload) {
+          void openImageFileInEditor(message.payload)
+        }
```

Comment:
[Q] Can we add `isImage` or similar to the telemetry event? This would mean we could quantify the split between the usage of each feature.